source/blog/2018-09-07-the-ember-times-issue-63.md

```diff
@@ -52,8 +52,15 @@ The time has come: The **Ember Community is starting its big move over to [Disco
 
 ---
 
-## [SECTION TITLE](#section-url)
+## New Computed Property RFCs Deprecating [volatile()](https://github.com/emberjs/rfcs/pull/370) and [readOnly()](https://github.com/emberjs/rfcs/pull/369) 2️⃣
+
+[Chris Garrett](https://github.com/pzuraq) proposed two new RFCs on [computed properties](https://guides.emberjs.com/release/object-model/computed-properties/). Both RFCs are centered around deprecating functions to adjust the defaults to what the common use cases developers face and to better align with native class syntax.
```

Comment:
Do you think it would be useful to link to the RFC for native classes in the last sentence if readers want to have more context on this?

E.g.

```
[...] and to better allign with  [native class syntax](https://github.com/emberjs/rfcs/pull/338).
```

Reply:
Definitely! added :)